### PR TITLE
Consolidate repositories/azureresourcegroup

### PR DESCRIPTION
**Pull request by IsaacCalligeros95**
PR to consolidate Sashimi.AzureResourceGroup Calamari halves into Calamari

### CODE REVIEW

File: source/Calamari.AzureResourceGroup/Calamari.AzureResourceGroup.csproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RootNamespace>Calamari.AzureResourceGroup</RootNamespace>
+    <AssemblyName>Calamari.AzureResourceGroup</AssemblyName>
+    <OutputType>Exe</OutputType>
```
Comment (IsaacCalligeros95) [inline source/Calamari.AzureResourceGroup/Calamari.AzureResourceGroup.csproj:5]: Note I had to remove ` <Nullable>enable</Nullable>` here to be able to build the project. Even with the setting enabled and all issues addressed the build fails with the following error:

`       "C:\Users\isaac\source\repos\Calamari\source\Calamari.AzureResourceGroup\Calamari.AzureResourceGroup.csproj" (build target) (2:2) ->
       (CoreCompile target) -> 
         CSC : error CS8630: Invalid 'nullable' value: 'Enable' for C# 7.3. Please use language version '8.0' or greater.`

Comment (IsaacCalligeros95) [reply source/Calamari.AzureResourceGroup/Calamari.AzureResourceGroup.csproj:7]: I'm still confused why this is happening but within the Calamari solution the build resolves 
`C:\Program Files\dotnet\sdk\5.0.406\Roslyn\bincore\csc.dll` with the following flag `/utf8output /deterministic+ /langversion:8`

In Sashimi.AzureResourceGroup this is resolved with `/utf8output /deterministic+ /langversion:default`, somehow this seems to resolve to C# 8 allowing the `<Nullable>enable</Nullable>` functionality to be used. Within the Calamari solution this resolved to C# 7.3 which makes sense given the net4.52 framework version.

Other projects within the Calamari repository specify the language version explicitly, `Calamari`, `Calamari.Common`, `Calamari.Shared`  etc. We've settled on following this approach but don't understand fully why it'

**Comment by IsaacCalligeros95** [comment]
Tested using the package locally - 
image

**Review by mjhilton — APPROVED** [review]
LGTM

Can you please pop the `dependency-consolidation` label on this PR?

**Pull request merged**